build/lint.js

```diff
@@ -38,13 +37,13 @@ function _spawn(proc, args, done) {
 }
 
 function _lintJS(files, done) {
-    _spawn('jshint', files, done);
+    _spawn((process.platform === 'win32') ? 'jshint.cmd' : 'jshint', files, done);
 }
 
 function _lintCSS(files, done) {
     var rules = JSON.parse(fs.readFileSync(_c.ROOT + ".csslintrc", "utf-8")),
         options = ["--errors=" + rules, "--format=compact", "--quiet"];
-    _spawn('csslint', files.concat(options), function (/*code*/) {
+    _spawn((process.platform === 'win32') ? 'csslint.cmd' : 'csslint', files.concat(options), function (/*code*/) {
```

Comment:
Just wondering if the platform returned when using https://msysgit.github.io/ is, as this could break it in there (since jshint and csslint are as in osx/linux land, if so?